agent/grpc-external/services/dataplane/get_envoy_bootstrap_params.go

```diff
@@ -73,7 +74,24 @@ func (s *Server) GetEnvoyBootstrapParams(ctx context.Context, req *pbdataplane.G
 		NodeId:      string(svc.ID),
 	}
 
-	bootstrapConfig, err := structpb.NewStruct(svc.ServiceProxy.Config)
+	// This is awkward because it's designed for different requests, but
+	// this fakes the ServiceSpecificRequest so that we can reuse code.
+	_, ns, err := configentry.MergeNodeServiceWithCentralConfig(
+		nil,
+		store.(configentry.StateStore),
```

Comment:
Ah, right. I had the cast because I was missing the `ReadResolvedServiceConfigEntries` from the `dataplane.StateStore` interface.

Fixed that, so no more cast